bpf/aya-bpf/src/maps/xdp/cpu_map.rs

```diff
@@ -0,0 +1,51 @@
+use core::{cell::UnsafeCell, mem};
+
+use aya_bpf_bindings::bindings::bpf_cpumap_val;
+
+use crate::{
+    bindings::{bpf_map_def, bpf_map_type::BPF_MAP_TYPE_CPUMAP},
+    helpers::bpf_redirect_map,
+    maps::PinningType,
+};
+
+#[repr(transparent)]
```

Comment:
Can we add docstrings here as well? Here and in methods.

I'm tring to cover already existing maps in #244 (still in progress, will try to finish this week). But in general, the goal is to have maps and the most important types documented also in aya-(e)bpf, so we can finally release it on crates.io in reasonable time.

Reply:
Done in 1fbb1d5f70096eb634816730b69882ff3039877c.